src/static/js/routes/careers/working-at-cfpb/index.js

```diff
@@ -0,0 +1,21 @@
+/* ==========================================================================
```

Comment:
I wasn't saying initialize everywhere, but it seems counter-intuitive to create duplicate inits on multiple index.js files when the same one could be used and just pulled into each page individually.